### ISSUE TITLE
Failed 4.5.3 build with PCH on arch linux

### ISSUE THREAD

**Issue by swhaat** [veyon/veyon#719]
Current Arch linux fails to build veyon 4.5.3 with the error:
```
make[2]: *** No rule to make target 'core/CMakeFiles/veyon-pch.dir/cmake_pch.hxx.gch', needed by 'core/CMakeFiles/veyon-pch.dir/veyon-pch_autogen/mocs_compilation.cpp.o'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:963: core/CMakeFiles/veyon-pch.dir/all] Error 2
```
also when the cmake build command includes `-DCMAKE_WITH_PCH=OFF`.

Veyon 4.5.2 builds without problems.

**Comment by kevinsmia1939** [comment]
I have the same problem on openSUSE Tumbleweed. 
But the flag is -DWITH_PCH:BOOL=OFF

**Comment by swhaat** [comment]
@kevinsmia1939 thanks for the correct flag.
If the flag is used, it builds correctly on arch.

**Comment by tobydox** [comment]
Thanks for reporting, the issue has been fixed in 5d01e560e9b549989f7ae5aabe9e51f57715f721.
